pubspec.yaml

```diff
@@ -1,24 +1,24 @@
 name: angular_components
-version: 0.9.0-alpha
+version: 0.9.0-alpha+1
 description: >
   The official Material Design components for AngularDart. Used at Google
   in production apps.
 homepage: https://webdev.dartlang.org/components
 author: Dart Team <misc@dartlang.org>
 environment:
-  sdk: '>=1.24.0 <2.0.0'
+  sdk: '>=2.0.0-dev.3.0 <2.0.0'
```

Comment:
It might be a good idea to mention the required SDK required lower bound in the README and/or changelog?

Reply:
Good idea. I'll add something.